libs/hdf-converters/src/aws-config-mapper.ts

```diff
@@ -0,0 +1,334 @@
+    }
+    return result;
+  }
+  private checkText(configRule: ConfigRule): string {
+    let params: any[] = [];
+    if (
+      configRule.InputParameters !== undefined &&
+      configRule.InputParameters !== '{}'
+    ) {
+      params = configRule.InputParameters.replace(/{/gi, '')
+        .replace(/}/gi, '')
+        .split(',');
+    }
+    const checkText = [];
+    checkText.push(`ARN: ${configRule.ConfigRuleArn || 'N/A'}`);
+    checkText.push(
+      `Source Identifier: ${configRule.Source?.SourceIdentifier || 'N/A'}`
+    );
+    if (params.length !== 0) {
+      checkText.push(`${params.join('<br/>').replace(/\"/gi, '')}`);
+    }
+    return checkText.join('<br/>');
+  }
+  private hdfDescriptions(configRule: ConfigRule) {
+    return [
+      {
+        data: this.checkText(configRule),
+        label: 'check'
+      }
+    ];
+  }
+  private getAccountId(arn: string): string {
+    const matches = arn.match(/:(\d{12}):config-rule/);
+    if (matches === null) {
+      return 'no-account-id';
+    } else {
+      return matches[0];
+    }
+  }
+  private async getControls(): Promise<ExecJSON.Control[]> {
+    let index = 0;
+    const controls = (await this.issues).map((issue: ConfigRule) => {
+      const control: ExecJSON.Control = {
+        id: issue.ConfigRuleId || '',
+        title: `${this.getAccountId(issue.ConfigRuleArn || '')} - ${
+          issue.ConfigRuleName
+        }`
+          .replace(/:/gi, '')
+          .replace(/config-rule/gi, ''),
+        desc: issue.Description || null,
+        impact: this.getImpact(issue),
+        tags: this.hdfTags(issue),
+        descriptions: this.hdfDescriptions(issue),
+        refs: [],
+        source_location: {ref: issue.ConfigRuleArn, line: 1},
+        code: '',
+        results: this.results[index]
+      };
+      index++;
+      return control;
+    });
+    return controls;
+  }
+  private getImpact(issue: ConfigRule): number {
+    if (_.get(issue, 'compliance') === 'NOT_APPLICABLE') {
+      return 0;
+    } else {
+      return 0.5;
+    }
+  }
+  public async toHdf(): Promise<ExecJSON.Execution> {
+    const hdf: ExecJSON.Execution = {
+      platform: {
+        name: 'Heimdall Tools',
+        release: HeimdallToolsVersion,
+        target_id: ''
+      },
+      version: HeimdallToolsVersion,
+      statistics: {
+        //aws_config_sdk_version: ConfigService., // How do i get the sdk version?
+        duration: null
+      },
+      profiles: [
+        {
+          name: NAME,
+          version: '',
+          title: NAME,
+          maintainer: null,
+          summary: NAME,
+          license: null,
+          copyright: null,
+          copyright_email: null,
+          supports: [],
+          attributes: [],
+          depends: [],
+          groups: [],
+          status: 'loaded',
+          controls: await this.getControls(),
+          sha256: ''
+        }
+      ]
+    };
+    return hdf;
+  }
```

Comment:
Should add a delay function to this file so that it can be used to avoid throttling exceptions from AWS.

```js
private delay(ms: number): Promise<void> {
  return new Promise((resolve) => setTimeout(resolve, ms));
}
```